source/user-manual/registering/index.rst

```diff
@@ -8,56 +8,118 @@ Registering agents
 .. meta::
   :description: Learn more about the different methods that can be used to register agents against the Wazuh manager.
 
-In this section, the registration process is described, and more specifically the different methods that users can use to register agents in the Wazuh manager.
+Wazuh agents communicate with the Wazuh server using outbound TCP/UDP connections to two different services:
 
-Prerequisites
--------------
+ - 1515/TCP: Service used for agents registration, to obtain a unique key. The communication is done over TLS. The obtained unique key (one per agent) is used later to authenticate with the Wazuh service (1514/TCP-UDP) and to encrypt traffic.
+ - 1514/TCP-UDP: Service used to send collected data and control messages.
 
-The registration service requires an SSL certificate on the manager in order to work. This certificate will be automatically generated by the package during the installation if the ``openssl`` package is installed. This certificate is stored in ``/var/ossec/etc/sslmanager.cert``.
+.. note::
 
-If you already have a certificate, you can use them just by copying them into the same path:
+    If the agent was deployed using :ref:`Deployment variables <deployment_variables>`, :ref:`Deployed with Ansible <wazuh_ansible>` or :ref:`Deployed with Puppet <wazuh_puppet>`, the registration process is different and described in their corresponding sections of the documentation.
```

Comment:
Can we explain here that if the deployment was done with variables, this section is completely unuseful?

Reply:
Done, hopefully it is better now.